core/lib/libtomcrypt/src/misc/mem_neq.c

```diff
@@ -74,6 +74,11 @@ int mem_neq(const void *a, const void *b, size_t len)
       ++pb;
    }
 
+   ret |= ret >> 4;
+   ret |= ret >> 2;
+   ret |= ret >> 1;
+   ret &= 1;
+
```

Comment:
@jenswi-linaro agreed, however, it seems like @tomstdenis prefer using the folding instead of double negation, see his comment in the PR Pascal links to in the commit message.